doc/quickstart.rst

```diff
@@ -0,0 +1,186 @@
+Charm++ Quickstart
+==================
+
+This section gives a concise overview of running your first Charm++ application.
+
+Installing Charm++
+------------------
+
+To download the latest Charm++ release, run:
+
+.. code-block:: console
+
+   $ wget http://charm.cs.illinois.edu/distrib/charm-6.9.0.tar.gz
+   $ tar xzf charm-6.9.0.tar.gz
+
+To download the development version of Charm++, run:
+
+.. code-block:: console
+
+   $ git clone https://github.com/UIUC-PPL/charm
+
+
+To build Charm++, use the following commands:
+
+.. code-block:: console
+
+   $ cd charm
+   $ ./build AMPI netlrts-linux-x86_64 --with-production -j4
+
+This is the recommenced version to install Charm++ on Linux systems.
```

Comment:
maybe say "to install Charm++ and AMPI"

Reply:
Thats a good point. Should we keep this to build AMPI, or change to charm++/LIBS?

Reply:
changed to charm++.